pkg/build/registry/buildlog/rest.go

```diff
@@ -125,10 +125,10 @@ func (r *REST) waitForBuild(ctx kapi.Context, build *api.Build) error {
 			}
 			switch obj.Status {
 			case api.BuildStatusCancelled:
-				errchan <- fmt.Errorf("Build %s/%s was cancelled", build.Namespace, build.Name)
+				errchan <- fmt.Errorf("Build %s/%s was cancelled. %s", build.Namespace, build.Name, buildutil.NoBuildLogsMessage)
```

Comment:
We shouldn't capitalize the name of our resources in error messages or UI
messages

On Jun 30, 2015, at 8:06 AM, Jakub Hadvig <notifications@github.com> wrote:

In pkg/build/registry/buildlog/rest.go
<https://github.com/openshift/origin/pull/3449#discussion_r33565181>:

> @@ -125,10 +125,10 @@ func (r *REST) waitForBuild(ctx kapi.Context, build *api.Build) error {
>  			}
>  			switch obj.Status {
>  			case api.BuildStatusCancelled:
> -				errchan <- fmt.Errorf("Build %s/%s was cancelled", build.Namespace, build.Name)
> +				errchan <- fmt.Errorf("Build %s/%s was cancelled. %s", build.Namespace, build.Name, buildutil.NoBuildLogsMessage)

@smarterclayton <https://github.com/smarterclayton> even if its the name of
the resource ?
eg:
https://github.com/openshift/origin/blob/master/pkg/build/generator/generator.go#L128

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3449/files#r33565181>.